bin/convert-themes-to-rgb.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
```

Comment:
This quick & dirty script is responsible for all the changes under `src/preset-themes` - I didn't manually edit any of them.

Reply:
oh good :)